tests/test_py_desc_converter.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+import unittest
+import pypandoc
+import mock
+import restructuredtext_lint
+import os
+
+from protoc_docs.bin import py_desc_converter
+from google.protobuf import descriptor_pb2 as desc
+
+curdir = os.path.realpath(os.path.dirname(__file__))
+
+
+class PythonDocsConversionTests(unittest.TestCase):
+    @mock.patch.object(pypandoc, 'convert_text')
+    def test_execute(self, convert_text):
+        convert_text.return_value = ''
+        descriptor_set = '%s/data/descriptor_set' % curdir
+        updated_desciprot_set = '%s/data/descriptor_set_updated_py_docs' % curdir
+        py_desc_converter.convert_desc(descriptor_set, updated_desciprot_set)
+        convert_text.assert_called()
+
+    @unittest.expectedFailure
+    def test_valid_rst(self):
+        descriptor_set = '%s/data/descriptor_set' % curdir
+        updated_desciprot_set = '%s/data/descriptor_set_updated_py_docs' % curdir
+        py_desc_converter.convert_desc(descriptor_set, updated_desciprot_set)
+
+        desc_set = desc.FileDescriptorSet()
+        with open(updated_desciprot_set, 'rb') as f:
+            desc_set.ParseFromString(f.read())
+
+        lint_errors = []
+        comment_count = 0
+        for comment in gather_comments_from_descriptor_set(desc_set):
```

Comment:
Nit: can do
```python
for count, comment in enumerate(gather_comments_from_descriptor_set(desc_set):
   ...

Reply:
when I do it the tests pass, but I get warning at L53 from intellij on line 53 saying that count may be  referenced before assignment. Is it idiomatic way to do (access enumerate count after the for loop)?

Reply:
Resolved AFK.